lib/SILOptimizer/IPO/LetPropertiesOpts.cpp

```diff
@@ -120,6 +127,16 @@ class InstructionsCloner : public SILClonerWithScopes<InstructionsCloner> {
 
 } // namespace
 
+// For debugging only.
```

Comment:
I  believe you'll get a unused-static-function warning if you compile in release mode. You could embed this function in an #ifdef NDEBUG.